libs/node-client/src/aggregate-instance.ts

```diff
@@ -35,6 +43,27 @@ export class AggregateInstance<
   }
 
   public async reload(): Promise<void> {
-    throw new Error('not implemented');
+    await this.mutex.runExclusive(async () => {
+      let version = this._version;
+      let id = this._id;
+
+      let events = await this.client.listAggregateEvents({
+        aggregate: {
+          id,
+          version,
+        },
+      });
+
+      events.map((event) => {
+        if (version < event.aggregate.version) {
+          this._id = event.aggregate.id;
+          this._version = event.aggregate.version;
+          this._state['root'] = {
```

Comment:
based on the event `type`, find from `eventHandlers` the appropriate event handler, then, calculate the new state